core/src/main/java/bio/terra/pearl/core/service/participant/EnrolleeService.java

```diff
@@ -279,6 +284,11 @@ public String generateShortcode() {
         return shortcode;
     }
 
+    @Transactional
+    public List<Enrollee> findGovernedEnrollees(List<EnrolleeRelation> enrolleeRelations) {
+        return findAll(enrolleeRelations.stream().map(EnrolleeRelation::getId).toList());
```

Comment:
BLOCKER: I dont' think this works as written
```suggestion
        return findAll(enrolleeRelations.stream().map(EnrolleeRelation::getEnrolleeId).toList());
```